integration-cli/docker_cli_build_test.go

```diff
@@ -4694,3 +4694,51 @@ func TestBuildDockerfileOutsideContext(t *testing.T) {
 
 	logDone("build - Dockerfile outside context")
 }
+
+func TestBuildMissingArgs(t *testing.T) {
+	// test to make sure these cmds w/o any args will generate an error
+	// Notice some commands are missing because its ok for them to
+	// not have any args - like: CMD, RUN, ENTRYPOINT
```

Comment:
@duglin 2 - 0 @tiborvass :)

I just wanna make sure we don't forget to update this test when we add new commands that's all. So maybe a comment in the dispatcher or somewhere else in the builder?